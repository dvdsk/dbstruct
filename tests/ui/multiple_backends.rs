use dbstruct::dbstruct;

#[dbstruct(db=sled, db=trait)]
struct Test {
    field: Option<u8>,
}

fn main() {}
