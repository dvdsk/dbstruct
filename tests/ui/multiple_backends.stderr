error: multiple database backends specified
 --> tests/ui/multiple_backends.rs:3:15
  |
3 | #[dbstruct(db=sled, db=trait)]
  |               ^^^^^^^^^^^^^^
  |
  = help: remove one of the backends
